source/database/mssql/scripts/dbscripts/PSP_PIMS_LATEST/Alter Up/115_DML_PIMS_CONSULTATION_TYPE.sql

```diff
@@ -0,0 +1,21 @@
+/* -----------------------------------------------------------------------------
+Populate the missing code values in the PIMS_CONSULTATION_TYPE table.
+. . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . 
+Author        Date         Comment
+------------  -----------  -----------------------------------------------------
+Doug Filteau  2023-Jan-30  Initial version
+----------------------------------------------------------------------------- */
+
+DELETE FROM PIMS_CONSULTATION_TYPE
+GO
+
+INSERT INTO PIMS_CONSULTATION_TYPE (CONSULTATION_TYPE_CODE, CONSULTATION_STATUS_TYPE_CODE, DESCRIPTION, OTHER_DESCRIPTION, DISPLAY_ORDER)
+VALUES
+  (N'1STNATION', N'UNKNOWN', N'First nation:',                '',                1),
```

Comment:
@dfilteau please remove the colons. The database should generally not include formatting like this. Formatting should be applied by the front-end.

Reply:
Yes, we need to help the BA's with best practices. Remember that they are non-technical resources and not DB design/API design experts so we should not be taking content from Confluence as-is without double checking everything and providing feedback.